02/Form.js

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
 
 class Form extends React.Component {
+    handleSubmit = e => {
+        e.preventDefault();
+        const name = e.target.elements.name.value;
+        this.props.onSubmit({name})
+       
+    }
     render() {
         return (
-            <section>
-                <input /><input type="submit" />
+            <section onSubmit={ this.handleSubmit} >
```

Comment:
Jeśli ma to być event `submit` to musimy tutaj mieć element `form` lub zrobić `onClick` dla "przycisku".